packages/apps/demo/src/Legend.tsx

```diff
@@ -131,7 +130,6 @@ export const Legend = (props: LegendProps) => {
           },
           content: {
             sx: {
-              pointerEvents: 'auto',
```

Comment:
Another unrelated-to-sharing change:

Moved restoration of pointer events deeper down in the tree. Turns out pointer events were being intercepted by the Drawer in the areas above/below the drawer content.